packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/ContentArrayGroup.stories.js

```diff
@@ -1,6 +1,5 @@
 import './index.scss';
-
-import { text, withKnobs } from '@storybook/addon-knobs';
+import { boolean, text, withKnobs } from '@storybook/addon-knobs';
```

Comment:
Filename should be renamed to `ContentGroup.stories.js`

Reply:
Okay.